clients/cloud/rust/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name    = "rust-kafka-client-example"
+version = "0.1.0"
+authors = ["GRECO, FRANK <frank@confluent.io>"]
+edition = "2018"
```

Comment:
2019

Reply:
that's not the current year, but the rust edition